ProfessorAssessor/Views/ReviewCell.m

```diff
@@ -0,0 +1,33 @@
+@import Parse;
+#import "ReviewCell.h"
+#import "Course.h"
+
+static NSDateFormatter *dateFormatter = nil;
+
+@implementation ReviewCell
+
+- (void)setReview:(Review *)review {
+    [review.course
```

Comment:
A review cell shouldn't be doing any fetching most likely. Can this information be pre-fetched in some way?